src/com/obelikova/console/calculator/DivisionByNullException.java

```diff
@@ -0,0 +1,11 @@
+package com.obelikova.console.calculator;
+
+/**
+ * Created by obielikova1 on 7/14/16.
+ */
+public class DivisionByNullException extends Exception {
```

Comment:
Was it an intentional wish to make a DivisionByNullException to be a checked exception?
The second thing is about the exception's name? Why it's called DivisionByNull if it's, in fact, is thrown when the second operand in division equals to zero?